crates/fire_dynamics_tools/src/chapter_2/equation_2_10.rs

```diff
@@ -0,0 +1,70 @@
+pub fn height_smoke_layer_interface(k: f64, q: f64, t: Vec<f64>, a_c: f64, h_c: f64) -> Vec<f64> {
+    t.iter().map(|&time| {
+        (((2.0 * k * q.powf(1.0/3.0) * time) / (3.0 * a_c)) + (1.0 / h_c.powf(2.0/3.0))).powf(-3.0/2.0)
+    }).collect()
```

Comment:
If you are confident in this equation and you have done the hand calcs for the tests and they are passing then I would say go for it. But I would be careful with this type of equation. It is VERY difficult to read and VERY easy to make a mistake. I would break it down into multiple parts and then rebuild the full equation to make it easier to read. i.e.
```suggestion
    t.iter().map(|&time| {
        let a = ((2.0 * k * q.powf(1.0/3.0) * time) / (3.0 * a_c))
        let b = (1.0 / h_c.powf(2.0/3.0))
        
        (a + b).powf(-3.0/2.0)
    }).collect()
```
I'm not asking for this to change. It is fine.